examples/meta/src/statistical_testing/linear_time_maximum_mean_discrepancy.sg

```diff
@@ -42,15 +42,15 @@ mmd.set_train_test_ratio(1)
 #![select_kernel_single]
 mmd.set_kernel_selection_strategy(enum EKernelSelectionMethod.KSM_MAXIMIZE_POWER)
 mmd.select_kernel()
-Kernel learnt_kernel_single = mmd.get_kernel()
+Kernel learnt_kernel_single = mmd.get("kernel")
 real width = learnt_kernel_single.get_real("log_width")
 #![select_kernel_single]
 
 #![select_kernel_combined]
 mmd.set_kernel_selection_strategy(enum EKernelSelectionMethod.KSM_MAXIMIZE_POWER, True)
 mmd.select_kernel()
-CombinedKernel learnt_kernel_combined = CombinedKernel:obtain_from_generic(mmd.get_kernel())
-RealVector weights = learnt_kernel_combined.get_subkernel_weights()
+Kernel learnt_kernel_combined = mmd.get("kernel")
+RealVector weights = learnt_kernel_combined.get_real_vector("subkernel_log_weights")
```

Comment:
@lambday I had to change the getter to the log weight field of `CCombinedKernel` as the method is non-const and therefore not exposable to the new `get` at the moment. Will this work to get the weights your framework is setting?